simulator/source/Combat_simulator.cpp

```diff
@@ -452,6 +454,39 @@ void Combat_simulator::mortal_strike(Sim_state& state)
     logger_.print("Current rage: ", int(rage));
 }
 
+void Combat_simulator::devastate(Sim_state& state)
+{
+    //TODO: Add DPR support
+    logger_.print("Devastate!");
+    int bonus = (config.exposed_armor && apply_delayed_armor_reduction) ? 0 : std::vector<int>{0, 35*2, 35*3, 35*4, 35*5, 35*5}[std::min(sunder_armor_stacks_, 5)];
+    double damage = ((state.main_hand_weapon.normalized_swing(state.special_stats) - state.special_stats.bonus_damage)/2 + bonus) * 1.10; //  devastate not affected by bonus damage
```

Comment:
The 110% was a misunderstanding of the magey github test with regards to the strange way devastate interacts with stacks of sunder armor. That post gives a formula for sunder armor damage with the 110% multiplier, but looking at it again the 110% was just to account for the fact that the test character had 1 hand weapon specialization and was a total damage calculation instead of just calculating the damage from devastate. Corrected this.